src/Rings/Laurent.jl

```diff
@@ -241,3 +241,12 @@ end
 
 
 
+function quo(R::Oscar.LaurentMPolyWrapRing, I::Oscar.LaurentMPolyIdeal)
```

Comment:
This should check that `I` is actually an ideal in `R`, e.g.:
```suggestion
function quo(R::Oscar.LaurentMPolyWrapRing, I::Oscar.LaurentMPolyIdeal)
  @req R === base_ring(I) "ring and ideal do not match"
```